src/pages/profile/Profile.css

```diff
@@ -0,0 +1,4 @@
+.profile_pic {
```

Comment:
I think a max-width and/or max-height would work better here. Any uploaded pictures will be exactly these dimensions otherwise.